x-pack/plugins/cases/public/components/markdown_editor/types.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FunctionComponent } from 'react';
+import { Plugin, PluggableList } from 'unified';
+// Remove after this issue is resolved: https://github.com/elastic/eui/issues/4688
```

Comment:
The issue seems to be resolved.

Reply:
see https://github.com/elastic/kibana/pull/97646#discussion_r620365295